frontend/app/route-helpers/renew-route-helpers.server.ts

```diff
@@ -24,6 +24,11 @@ export interface RenewState {
     socialInsuranceNumber: string;
   };
   readonly maritalStatus?: string;
+  readonly contactInformation?: {
+    phoneNumber?: string;
+    phoneNumberAlt?: string;
+    email?: string;
+  };
```

Comment:
The contact information state should be left out of this PR as it's handled in #2246.